content/fileOptions.Rmd

```diff
@@ -42,325 +43,1286 @@ opts_chunk$set(
 )
 ```
 
-In the group that I work with [https://owi.usgs.gov/datascience/](https://owi.usgs.gov/datascience/), the vast majority of the projects use flat files for data storage. Sometimes, the files get a bit large, so we create a set of files...but basically we've been fine without wading into the world of databases. Recently however, the data involved in our projects are creeping up to be bigger and bigger. We're still not anywhere in the "BIG DATA (TM)" realm, but big enough to warrant exploring options. This post explores the options: csv (both from `readr` and `data.table`), RDS, fst, sqlite, feather, monetDB. One of the takeaways I've learned was that there is not a single right answer. This post will attempt to lay out the options and summarize the pros and cons.
-
-In a blog post that laid out similar work:  [sqlite-feather-and-fst](https://kbroman.org/blog/2017/04/30/sqlite-feather-and-fst/) and continued [here](https://kbroman.org/blog/2017/05/11/reading/writing-biggish-data-revisited/), Karl Broman discusses his journey from flat files to "big-ish data". I've taken some of his workflow, added more robust for `fst` and `monetDB`, and used my own data.
+# TLDR!
+
+This blog post discusses the thought process for picking out file formats and when/if to use a database. It was interesting for me to write, I learned a lot, but if I'm honest...you are just here for the final table. I get that, your time is precious. So, here's the final results. If you want the longer explanations or the code examples that generated the table...continue reading. The table lists all of the packages that were tested, and the time it took to read (in a few ways) data, the time it took to write the data, and the file size. The timing was measured using the `microbenchmark` package. The R code used to generate equivalent data frames within is included for each package.
+
+
+<table class="table table-striped table-bordered" style="margin-left: auto; margin-right: auto;">
```

Comment:
You can remove this classes, because USWDS have them. But if you wanted stripes you could add something like this:

https://github.com/stphnwlkr/USWDS-Extended/blob/master/_table.scss#L12-L16